solutions/observability/observability-ai-assistant.md

```diff
@@ -202,6 +202,20 @@ After creating the pipeline, complete the following steps:
 
     Ask something to the AI Assistant related with the indexed data.
 
+### Add user-specific system prompts
+
+User-specific prompts customize how the AI assistant responds by appending personalized instructions to all of your system prompts. For example, you could specify "Always respond in French," and all subsequent responses will be in French.
+
+A user-specific prompt only applies to the user that sets it.
+
+% Do we have any other suggestions on how this could be useful for users?
```

Comment:
Instructions are very useful if they want to configure specific workflows. I think @davidgeorgehope had some concrete examples